golem_messages/datastructures.py

```diff
@@ -68,13 +68,35 @@ def __setitem__(self, key, value):
         validator = getattr(
             self, 'validate_{}'.format(key), None
         )
-
         if callable(validator):
             validator(value=value)  # noqa pylint:disable=not-callable
 
         super().__setitem__(key, value)
 
 
+class ValidatingDependentDictItems(ValidatingDict):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.validate_dependent_dict_items()
+
+    def validate_dependent_dict_items(self):
+        dependent_items_list = getattr(self, 'DEPENDENT_ITEMS', [])
```

Comment:
@kbeker wouldn't it be better to verify if `DEPENDENT_ITEMS` is set in the `__init__` and initialize it there to an empty list if it's not? then you wouldn't need this check here and, as a bonus, you wouldn't need to escape pylint's complaint about `DEPENDENT_ITEMS` not being an iterable below...

Reply:
I have change it just to: 
```python
for items in getattr(self, 'DEPENDENT_ITEMS', []):
```
It doesn't give warning any more and works as it should work.